src/main/java/uk/gov/digital/ho/hocs/casework/api/utils/ActionDataHelpers.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.digital.ho.hocs.casework.api.utils;
+
+import lombok.extern.slf4j.Slf4j;
+import uk.gov.digital.ho.hocs.casework.domain.model.ActiveStage;
+import uk.gov.digital.ho.hocs.casework.domain.model.CaseData;
+
+import java.time.LocalDate;
+import java.util.Map;
+
+@Slf4j
+public class ActionDataHelpers {
+    private ActionDataHelpers() {}
+
+    // COPIED FROM CaseDataService to avoid cyclic dependency.
```

Comment:
If it's an exact copy can CaseDataService use this class aswell?

Reply:
Better yet maybe a CaseDeadlineHelper that they both use?

Reply:
Ok... so when have looked at this, the method is somewhat different now from when it was originally taken from the `CaseDataService` (August last year). So I've renamed the method to distinguish and changed the class name to  Liam's suggestion.

Reply:
I've also removed some logic that is not required for either Extensions or Suspensions.